CDP4ParameterSheetGenerator/RowModels/OptionSheet/NestedElementExcelRow.cs

```diff
@@ -91,6 +103,9 @@ private void ProcessNestedParameters(IEnumerable<NestedParameter> nestedParamete
                     }
 
                     var nestedParameterExcelRow = new NestedParameterExcelRow(nestedParameter);
+                    nestedParameterExcelRow.Level = this.Level + 1;
```

Comment:
This works ok, but shouldn't the NestedParameterExcelRow class itself be responsible for adding spaces to its name?

Reply:
Well, I think here this is ok. NestedParameterExcelRow in principle should never exist outside of a NestedElement and its level should be directly tied to the said element.